apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -75,8 +75,8 @@ export const SwapCommitButton = memo(function SwapCommitButton({
     [Field.INPUT]: { currencyId: inputCurrencyId },
     [Field.OUTPUT]: { currencyId: outputCurrencyId },
   } = useSwapState()
-  const inputCurrency = useCurrency(inputCurrencyId)
-  const outputCurrency = useCurrency(outputCurrencyId)
+  const inputCurrency = useCurrency(inputCurrencyId) as Currency
```

Comment:
why are we using type assertion here?

Reply:
no specific reason i can remove. i think i added ages back but yep should be removed